src/backend/joanie/core/serializers/admin.py

```diff
@@ -1269,6 +1271,61 @@ def get_owner_name(self, instance) -> str:
         return instance.owner.get_full_name() or instance.owner.username
 
 
+class AdminOrderExportSerializer(serializers.ModelSerializer):
+    """
+    Read only light serializer for Order export.
+    """
+
+    class Meta:
+        model = models.Order
+        fields_labels = (
+            ("id", "Référence de commande"),
+            ("created_on", "Date de création"),
+            ("owner_name", "Propriétaire"),
+            ("total", "Prix"),
+        )
+        fields = [field for field, label in fields_labels]
+        read_only_fields = fields
+
+    @property
+    def headers(self):
+        """
+        Return the headers of the CSV file.
+        """
+        return [label for field, label in self.Meta.fields_labels]
+
+    owner_name = serializers.SerializerMethodField(read_only=True)
+
+    def get_owner_name(self, instance) -> str:
+        """
+        Return the full name of the order's owner if available,
+        otherwise fallback to the username
+        """
+        return instance.owner.get_full_name() or instance.owner.username
```

Comment:
We are using this logic in several place, what do you think to make an util ?

Reply:
I moved it to a user model property. WDYT ?